main.js

```diff
@@ -616,3 +627,88 @@ function prevb(){
 grab("fl").onclick = flashlight;
 grab("prevb").onclick = prevb;
 grab("nextb").onclick = nextb;
+
+/* removes all courses from the courses list. */
+function rmcourses(){
+	grabclass("courses")[0].innerHTML = '<div style="text-align: center; margin: 0.8em;"> current courses </div>';
+	courses = [];
+}
+
+function idiot(isLoad){
+	isLoad = typeof isLoad !== 'undefined' ? isLoad : false;
+	var tempstate = getstate();
+	if(tempstate !== {}) {
+		tempstate.i = 1;
+	}
+	if(!isLoad) {
+		rmcourses();
+		rmblocks();
+		courses = [];
+	}
+	dontfill_color = "#000";
+	palette = ["#CC0000", "#7A00CC", "#29A329", "#CCCC00",  "#00CCCC", "#00008A", "#002900", "#E62EB8", "#005C5C", "#CC3300", "#808080", "#00FF00", "#666633", "#002E2E"];
+	var elButtons = document.querySelectorAll('button');
+	for(i = 0; i < elButtons.length; i++) {
+  		elButtons[i].classList.add('button-idiot');
```

Comment:
I would think something like
```js
all = document.querySelectorAll("*");
for(i=0; i<all.length; i++) all[i].classList.add("idiot");
```
would work better? Then you can style with
```css
.courses.idiot {
    background-color...
}
```

Reply:
Thanks for the input, now it's way better!